C/jets.c

```diff
@@ -655,6 +655,38 @@ RIGHT_SHIFT_(4,16)
 RIGHT_SHIFT_(8,32)
 RIGHT_SHIFT_(8,64)
 
+#define ROTATE_(log, bits)                                                                     \
+/* Precondition: 0 <= amt < bits */                                                            \
```

Comment:
nit: could be asserted at runtime

Reply:
Normally I don't add asserts to jets.

We cannot both have policy of keeping asserts in production and putting asserts in code that is to be executed as quickly as possible.

That said, I see I do use `simplicity_debug_assert` at one point, which is an assert that is removed when `PRODUCTION`, so I suppose I could use that if you think I should.

Reply:
I think I would prefer not to do this.  If anything I think I would prefer to remove the existing `simplicity_debug_assert` that is used elsewhere.